config/v201/component_schemas/standardized/InternalCtrlr.json

```diff
@@ -449,6 +449,22 @@
           "default": "5",
           "type": "integer"
       },
+      "VariableMonitoringProcessTime": {
```

Comment:
nit: `PeriodicMonitorsProcessingInterval` seems more appropriate to me

Reply:
It can also process the alert ones, in case there is a 'unprocessed' queue (some internal variable modifications corner cases), so it applies for both in the end. Are you sure?

Reply:
ok :+1: